### PR TITLE
[IRRC-79] Refactor createstudent to vuex

### PR DESCRIPTION
**Pull request by dzldebby**

### CODE REVIEW

File: frontend/src/store/students.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const MUTATIONS = Object.freeze({
+    SET_STUDENTS: 'SET_STUDENTS'
+  })
+
+
+
```
Comment (thchia) [inline frontend/src/store/students.js:6]: Can we make these large spaces only 1 line high?

**Review by thchia — APPROVED** [review]

**Review by dzldebby — COMMENTED** [review]
Hmm I got a 500 on PATCH. The error was: 
image

Tried to re-run db:migrate and db:seed, and also tried to edit Native Language in the PATCH but doesn't work. Do we need a NativeLanguageString somewhere in the backend?

**Comment by thchia** [comment]
@dzldebby When you did the patch did you pass in native language id or native language string?

**Comment by dzldebby** [comment]
> Hmm I got a 500 on PATCH. The error was:
> image
> 
> Tried to re-run db:migrate and db:seed, and also tried to edit Native Language in the PATCH but doesn't work. Do we need a NativeLanguageString somewhere in the backend?

Please ignore this comment, it was supposed to be for IRRC-50 instead.

**Review by nondescryptid — APPROVED** [review]
Tested and it works! LGTM 👍

**Pull request merged**